app/src/main/res/layout/fragment_plant_details.xml

```diff
@@ -24,26 +24,4 @@
         android:id="@+id/gridlayout">
```

Comment:
Shouldn't this file be completely deleted? There should be no PlantDetailsFragment right?

Reply:
I also can't see that the fragment is deleted

Reply:
No it should be there, it just should not show the details for the MyPlant part - we are still using it for the lexicon